
Showing logs for build [32mmeta-seldonio-seldon-core-pr-20-gn7v6-17[0m stage [32mmeta-pipeline[0m and container [32mstep-credential-initializer-kk4mq[0m
{"level":"info","ts":1600279144.5703382,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1600279144.571148,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-20-gn7v6-17[0m stage [32mmeta-pipeline[0m and container [32mstep-working-dir-initializer-2z8mq[0m
{"level":"info","ts":1600279146.0129597,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1600279146.159179,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-20-gn7v6-17[0m stage [32mmeta-pipeline[0m and container [32mstep-place-tools[0m

Showing logs for build [32mmeta-seldonio-seldon-core-pr-20-gn7v6-17[0m stage [32mmeta-pipeline[0m and container [32mstep-git-source-meta-seldonio-seldon-core-pr-20-gn7v6-9pd8b[0m
{"level":"info","ts":1600279153.8985999,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1600279160.3625848,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ 8a6da8949a118f211d8970139b8235b79d02e22d in path /workspace/source"}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-20-gn7v6-17[0m stage [32mmeta-pipeline[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mmeta-seldonio-seldon-core-pr-20-gn7v6-17[0m stage [32mmeta-pipeline[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:8a6da8949a118f211d8970139b8235b79d02e22d,2023:f3bf6449f4ad3e228201d48eb3c1224aea04650e
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git fetch origin --unshallow f3bf6449f4ad3e228201d48eb3c1224aea04650e: 8a6da8949a118f211d8970139b8235b79d02e22d:
DEBUG: ran git fetch --unshallow origin f3bf6449f4ad3e228201d48eb3c1224aea04650e: 8a6da8949a118f211d8970139b8235b79d02e22d: in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git branch
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git checkout master
DEBUG: ran git checkout master in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git reset --hard 8a6da8949a118f211d8970139b8235b79d02e22d
DEBUG: ran git reset --hard 8a6da8949a118f211d8970139b8235b79d02e22d in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git clean -fd .
DEBUG: ran clean --force -d . in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git merge f3bf6449f4ad3e228201d48eb3c1224aea04650e
error: error during merge: merging f3bf6449f4ad3e228201d48eb3c1224aea04650e into master: git output: Auto-merging operator/controllers/seldondeployment_controller.go
Auto-merging executor/cmd/executor/main.go
CONFLICT (content): Merge conflict in executor/cmd/executor/main.go
Removing examples/models/nvidia-mnist/tensorrt_mnist/mnist_labels.txt
Removing examples/models/nvidia-mnist/tensorrt_mnist/config.pbtxt
Removing examples/models/nvidia-mnist/nvidia_mnist.ipynb
Removing examples/models/nvidia-mnist/nvidia-mnist/values.yaml
Removing examples/models/nvidia-mnist/nvidia-mnist/templates/mnist_nvidia_deployment.json
Removing examples/models/nvidia-mnist/nvidia-mnist/Chart.yaml
Removing examples/models/nvidia-mnist/fetch-model.sh
Removing examples/models/nvidia-mnist/MnistTransformer.py
Removing examples/models/nvidia-mnist/Makefile
Removing examples/models/nvidia-mnist/.s2i/environment
Removing doc/source/examples/nvidia_mnist.nblink
Automatic merge failed; fix conflicts and then commit the result.: failed to run 'git merge f3bf6449f4ad3e228201d48eb3c1224aea04650e' command in directory '', output: 'Auto-merging operator/controllers/seldondeployment_controller.go
Auto-merging executor/cmd/executor/main.go
CONFLICT (content): Merge conflict in executor/cmd/executor/main.go
Removing examples/models/nvidia-mnist/tensorrt_mnist/mnist_labels.txt
Removing examples/models/nvidia-mnist/tensorrt_mnist/config.pbtxt
Removing examples/models/nvidia-mnist/nvidia_mnist.ipynb
Removing examples/models/nvidia-mnist/nvidia-mnist/values.yaml
Removing examples/models/nvidia-mnist/nvidia-mnist/templates/mnist_nvidia_deployment.json
Removing examples/models/nvidia-mnist/nvidia-mnist/Chart.yaml
Removing examples/models/nvidia-mnist/fetch-model.sh
Removing examples/models/nvidia-mnist/MnistTransformer.py
Removing examples/models/nvidia-mnist/Makefile
Removing examples/models/nvidia-mnist/.s2i/environment
Removing doc/source/examples/nvidia_mnist.nblink
Automatic merge failed; fix conflicts and then commit the result.'
[31m
Pipeline failed on stage 'meta-pipeline' : container 'step-git-merge'. The execution of the pipeline has stopped.[0m
